lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1082,6 +1082,7 @@ def cla(self):
         # Disabling mouse interaction might have been needed a long
         # time ago, but I can't find a reason for it now - BVR (2012-03)
         #self.disable_mouse_rotation()
+        Axes.cla(self)
```

Comment:
I once tried super-fying Axes3D (about 3 years ago). Some weird bugs started happening, so I dropped it. At the time, I was highly inexperienced with super(), so I might have been doing it wrong.

I am a firm believer in keeping PRs orthogonal. If you want to take a crack at super-fying Axes3D, you are more than welcome to try, but do so in a separate PR.